C-Plus-Plus/cp/min_circles_to_be_removed.cpp

```diff
@@ -0,0 +1,70 @@
+/*Problem Statement:
+Given n Circles present in x-y plane such that all the circles have their centre aligned on the x-axis.
+The task is to remove some of them, such that no two circles are intersecting. 
+Find the minimum number of circles that need to be removed. */ 
+
+#include <bits/stdc++.h>
+using namespace std;
+
+/* this function will help in sorting the circles on the 
+basis of their radius */
+bool compare(pair<int, int> a, pair<int, int> b)
+{
+    if ((a.first + a.second) == (b.first + b.second))
+    {
+        return (a.first - a.second) < (b.first - b.second);
+    }
+    else
+    {
+        return (a.first + a.second) < (b.first + b.second);
+    }
+}
+
+int main()
+{
+    int ans = 1;
+    int n, c, r;
```

Comment:
Use proper naming conventions don't use single letter and also while reading inputs use cout for display proper message